pkg/pipelines/utils_test.go

```diff
@@ -113,6 +118,64 @@ func TestPushRepository(t *testing.T) {
 	e.assertCommandsExecuted(t, want)
 }
 
+func TestPushRepositoryWithRemoteAdd(t *testing.T) {
+	repo := "git@github.com:testing/testing.git"
+	opts := &BootstrapOptions{
+		OutputPath: "/tmp",
+	}
+	outputs := [][]byte{
+		[]byte("Initialized empty Git repository in /tmp/.git/"),
+		[]byte(""),
+	}
+	e := newMockExecutor(outputs...)
+	testErr := errors.New("test error")
+	e.errors.push(testErr)
+	e.errors.push(nil)
+	e.errors.push(nil)
+	e.errors.push(nil)
+	e.errors.push(nil)
```

Comment:
Duplicate lines, no ?

Reply:
Need to push the error messages as nil to the stack so that it gives required error only for the targeted check..